encryptx-frontend/src/app/components/forms/decrypt-form.tsx

```
@@ -0,0 +1,413 @@
+        }
+      }
+
+      xhr.onerror = () => reject(new Error("Network error"))
+
+      try {
+        const buffer = await file.arrayBuffer()
+        xhr.send(buffer)
+      } catch (error) {
+        reject(new Error("Failed to read file"))
+      }
+    })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)